src/smt/set_defaults.cpp

```diff
@@ -89,8 +89,7 @@ void setDefaults(LogicInfo& logic, bool isInternalSubsolver)
     // set this option if the input format is SMT LIB 2.6. We also set this
     // option if we are sygus, since we assume SMT LIB 2.6 semantics for sygus.
     options::bitvectorDivByZeroConst.set(
-        language::isInputLang_smt2_6(options::inputLanguage())
-        || language::isInputLangSygus(options::inputLanguage()));
+        !language::isInputLang_smt2_5(options::inputLanguage(), true));
```

Comment:
Since 'true' is the default for this option, shouldn't this be **disabled** for smt-lib 2.5?

Reply:
Maybe I am missing something but the intention is to set the option to `false` when the input language is SMT-LIB 2.5 and `true` if it is any other language.